popups.css

```
@@ -6,8 +6,16 @@
     margin-top: 0.5rem;
 }
 .lsp_popup hr {
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
     border-color: color(var(--foreground) alpha(0.10));
 }
+.lsp_popup h1 { font-size: 1rem; }
+.lsp_popup h2 { font-size: 1rem; }
+.lsp_popup h3 { font-size: 1rem; }
+.lsp_popup h4 { font-size: 1rem; }
+.lsp_popup h5 { font-size: 1rem; }
+.lsp_popup h6 { font-size: 1rem; }
```

Avoid repeating yourself
```suggestion
.lsp_popup h1,
.lsp_popup h2,
.lsp_popup h3,
.lsp_popup h4,
.lsp_popup h5,
.lsp_popup h6 {
    font-size: 1rem;
}
```